### PR TITLE
Create zh-cn-README.md

### PR DESCRIPTION
**Pull request by HelloGwkki**
中文README.md！
CHINESE README.md!

### CODE REVIEW

**Comment by HelloGwkki** [comment]
Moooed

File: zh-cn-README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## 安装需求
+- Python 3.6或更高版本。
+
+## 安装
+- 使用管理员权限打开CMD。
+- 输入以下命令：`pip install neofetch-win`。
+- 现在你可以在CMD输入`neofetch`来查看结果。
+
+### 可用的颜色
+黑色，红色，绿色，黄色，蓝色，洋红色，青色，白色。
```
Comment (AlexFlipnote) [inline zh-cn-README.md]: Could you add that the colours has to be written in English?

Comment (HelloGwkki) [reply zh-cn-README.md]: okay

Comment (HelloGwkki) [reply zh-cn-README.md]: please wait

Comment (HelloGwkki) [reply zh-cn-README.md]: done!

File: zh-cn-README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Python 3.6或更高版本。
+
+## 安装
+- 使用管理员权限打开CMD。
+- 输入以下命令：`pip install neofetch-win`。
+- 现在你可以在CMD输入`neofetch`来查看结果。
+
+### 可用的颜色
+黑色，红色，绿色，黄色，蓝色，洋红色，青色，白色。(black,red, green, yellow, blue, magenta, cyan, white)
+#### 译者提示：当然，您必须使用英语输入。
```
Comment (HelloGwkki) [inline zh-cn-README.md:19]: add english

**Comment by AlexFlipnote** [comment]
👍

**Pull request merged**